dcos/marathon.py

```diff
@@ -556,6 +565,23 @@ def get_task(self, task_id):
 
         return task
 
+    def get_app_schema(self):
+        """Returns app json schema
+
+        :returns: application json schema
+        :rtype: json schema or None if endpoint doesn't exist
+        """
+
+        version = self.get_version()
+        schema_version = LooseVersion("0.9.0")
+        if version < schema_version:
```

Comment:
OK.  Created [DCOS-2117](https://mesosphere.atlassian.net/browse/DCOS-2117) to track the test environment updates that need to be done.